internal/rest/dequeue_nodes.go

```diff
@@ -288,6 +288,12 @@ func (rest *RestOperations) CheckAndPublishForRetry(err error, publishKey avicac
 					rest.PublishKeyToSlowRetryLayer(publishKey, key)
 					return true
 				}
+			case 412:
```

Comment:
should we make it more generic, for all objects?

Reply:
Don't think it is needed for other objects. From rest layer 412 is seen only for VSVIP (Had done a search on Concurrent update in FT logs)